app/editor/src/features/navbar/NavBar.tsx

```diff
@@ -1,38 +1,43 @@
-import React from 'react';
-import { useLocation } from 'react-router-dom';
-import { Claim } from 'tno-core';
+import React, { useState } from 'react';
+import { Claim, NavBarGroup, NavBarItem, Show } from 'tno-core';
 import { Row } from 'tno-core/dist/components/flex/row';
-import { NavBarGroup, NavBarItem } from 'tno-core/dist/components/navbar';
 
 /**
  * The navigation bar that is used throughout the TNO editor application. Add or remove navigation bar items here.
  */
 export const NavBar: React.FC = () => {
-  const location = useLocation();
+  const [activeHover, setActiveHover] = useState<'admin' | 'editor' | ''>('');
+  console.log(activeHover);
 
-  const showEditor =
-    location.pathname.startsWith('/contents') ||
-    location.pathname.startsWith('/storage') ||
-    location.pathname.startsWith('/combined');
-  const showAdmin = location.pathname.startsWith('/admin');
-  const [activeHover, setActiveHover] = React.useState(false);
   return (
-    <div onMouseOver={() => setActiveHover(true)} onMouseLeave={() => setActiveHover(false)}>
+    <div onMouseLeave={() => setActiveHover('')}>
       <NavBarGroup className="navbar">
         <Row>
-          <NavBarItem navigateTo="/contents" label="Editor" claim={Claim.editor} />
-          <NavBarItem navigateTo="/admin" label="Admin" claim={Claim.administrator} />
+          <div className="editor" onMouseOver={() => setActiveHover('editor')}>
+            <NavBarItem
+              activeHoverTab={activeHover}
```

Comment:
Is the `activeHoverTab` prop to keep track of which parent is active?

Reply:
Correct